test/controllers/admin/users_controller_test.rb

```diff
@@ -87,6 +87,7 @@
 
   describe 'a DELETE to #destroy' do
     let(:user) { users(:viewer) }
+    let!(:locks) { Array.new(2, Lock.create!(user: users(:viewer), stage: stages(:test_staging))) }
```

Comment:
this does not do what you want `Array.new(2, :x) -> [:x, :x]`
also you cannot have 2 locks on the same stage